paasta_tools/marathon_tools.py

```diff
@@ -227,7 +227,7 @@ def get_autoscaling_params(self):
         default_params = {
             'method': 'default',
         }
-        return self.config_dict.get('autoscaling', default_params)
```

Comment:
I mean, ship the kwargs method as long as it works as intended.

Maybe be sure to update the docs?